lib/v2/bilibili/cache.js

```diff
@@ -218,7 +218,39 @@ module.exports = {
                 ).data
         );
         const $ = cheerio.load(data);
-        const description = $('#read-article-holder').html();
+        let description = $('#read-article-holder').html();
+        if (!description) {
+            try {
+                const newFormatData = JSON.parse(
+                    $('script:contains("window.__INITIAL_STATE__")')
+                        .text()
+                        .match(/window\.__INITIAL_STATE__\s*=\s*(.*?);\(/)[1]
+                );
+
+                if (newFormatData && newFormatData.readInfo && newFormatData.readInfo.opus && newFormatData.readInfo.opus.content && newFormatData.readInfo.opus.content.paragraphs) {
+                    description = '';
+                    for (const element of newFormatData.readInfo.opus.content.paragraphs) {
+                        if (element.para_type === 1) {
+                            for (const text of element.text.nodes) {
+                                if (text && text.word && text.word.words) {
+                                    description += `<p>${text.word.words}</p>`;
+                                }
+                            }
+                        }
+                        if (element.para_type === 2) {
+                            for (const image of element.pic.pics) {
+                                description += `<p class="normal-img" contenteditable="false"><img src="${image.url}@progressive.webp" data-status="loaded" alt="read-normal-img" data-w="${image.width}" data-h="${image.height}"></p>`;
+                            }
+                        }
+                        if (element.para_type === 3) {
+                            description += `<figure class="img-box" contenteditable="false"><img class="cut-off-5" src="${element.line && element.line.pic && element.line.pic.url}" referrerpolicy="no-referrer"></figure>`;
```

Comment:
Please remove useless class name and attributes.

No need to add `referrerpolicy` since RSSHub will always [do it for you](https://github.com/DIYgod/RSSHub/blob/cdc36d3554789bce4d7e457906f4871f08c11a54/lib/middleware/parameter.js#L144).